src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.commands.Command;
+
+/**
+ * Represents the chatbot Duke.
+ */
```

Comment:
Public classes must have `@author` and `@version` in the JavaDoc comments under the main description